pkgs/by-name/ge/geo/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  testers,
+  geo
+}:
+
+buildGoModule rec {
+  pname = "geo";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "MetaCubeX";
+    repo = "geo";
+    rev = "v${version}";
+    hash = "sha256-lwFBevf3iP90LgnfUqweCjPBJPr2vMFtRqQXXUC+cRA=";
+  };
+
+  vendorHash = "sha256-FXvuojlMZRzi8TIQ2aPiDH7F9c+2dpe4PYzYWljfUIc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.version = testers.testVersion { package = geo; };
```

```suggestion
    tests.version = testers.testVersion {
      package = geo;
      command = "${lib.getExe geo} --help";
      version = "v${version}";
    };
```
it looks like `geo` doesn't actually have a `--version` argument. we can get it from the output of `--help` instead, though